src/__tests__/decide.js

```diff
@@ -25,6 +25,7 @@ describe('Decide', () => {
     given('decide', () => new Decide(given.posthog))
     given('posthog', () => ({
         config: given.config,
+        debug: true,
```

Comment:
is this needed given we set the Window value too?

Reply:
Good point